example/docker-compose/etc/otel-collector.yaml

```diff
@@ -0,0 +1,15 @@
+receivers:    
+  jaeger:     
+    protocols:
+      thrift_http:
+exporters:
+  otlp:
+    endpoint: tempo:55680
+    insecure: true
+    headers:
+      x-scope-orgid: authed-tenant
+service:
+  pipelines:
```

Comment:
There is some ongoing discussion on label based routing. In that case, is it possible to create multiple pipelines, one for every tenant like so:

```
exporters:
  otlp-1:
    endpoint: tempo:55680
    headers:
      x-scope-orgid: authed-tenant-1
  otlp-2:
    endpoint: tempo:55680
    headers:
      x-scope-orgid: authed-tenant-2

processor:
  <label-based-routing-processor>

service:
  pipelines:
    traces-1:
      receivers: [jaeger]
      processors: <label-based-routing-processor>
      exporters: [otlp-1] 
    traces-2:
      receivers: [jaeger]
      processors: <label-based-routing-processor>
      exporters: [otlp-2] 
```

..that can give us real multitenant behaviour?

Reply:
You can already do it with the filter processor:  https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/filterprocessor

Although you'd have to be careful with your filters and tags so you didn't drop any spans.

Reply:
Can we document this option as well?

Reply:
did some digging and sadly the filter processor only applies to metrics.  perhaps it will be extended at some point and we can document.